docs/develop/power-automate-integration.md

```diff
@@ -56,7 +56,7 @@ When adding input parameters to a script's `main` function, consider the followi
 1. Object types are allowed if they contain properties of type `string`, `number`, `boolean`, supported arrays, or other supported objects. The following example shows nested objects that are supported as parameter types:
 
     ```TypeScript
-    // Office Scripts can return an Employee object because Position only contains strings and numbers.
+    // Office Scripts can use an Employee object because Position only contains strings and numbers.
```

Comment:
Saying it works because it only contains strings and numbers suggests to me that you can only use strings and numbers for this scenario